tests/user.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from '@playwright/test';
+import { api } from './utils';
+import * as config from '../init/config';
+
+let ideaId = 0;
+
+test.describe('/api/users', () => {
+
+    test("should show user me", async ({ request }) => {
```

Comment:
Can we also test that I can't get data of another user?

Reply:
Sure.

Reply:
@chungthuang 
I think we have a bug here. In production we allow findOne and update for users collection for authenticated role. And there's no filtering on who can do what. Effectively, I was able to update other user's interest using my browser session and manipulating request URL.

Reply:
This sounds like a bug to me as well. Can we create a bug issue?

Reply:
Will add one.